Library/ViewModels/SettingsDeleteAccountCellViewModel.swift

```diff
@@ -0,0 +1,46 @@
+import Foundation
+import KsApi
+import Prelude
+import ReactiveSwift
+import ReactiveExtensions
+import Result
+
+public protocol SettingsDeleteAccountCellViewModelInputs {
+  func configureWith(user: User)
+  func deleteAccountTapped()
+}
+
+public protocol SettingsDeleteAccountCellViewModelOutputs {
+  var notifyDeleteAccountTapped: Signal<URL, NoError> { get }
+}
+
+public protocol SettingsDeleteAccountCellViewModelType {
+  var inputs: SettingsDeleteAccountCellViewModelInputs { get }
+  var outputs: SettingsDeleteAccountCellViewModelOutputs { get }
+}
+
+public final class SettingsDeleteAccountCellViewModel: SettingsDeleteAccountCellViewModelType,
+  SettingsDeleteAccountCellViewModelInputs, SettingsDeleteAccountCellViewModelOutputs {
+
+  public init() {
+    self.notifyDeleteAccountTapped = self.deleteAccountTappedProperty.signal
+      .map {
+        AppEnvironment.current.apiService.serverConfig.webBaseUrl.appendingPathComponent("/profile/destroy")
```

Comment:
I'd just chain these:

```swift
AppEnvironment.current.apiService.serverConfig.webBaseUrl
  .appendingPathComponent("profile")
  .appendingPathComponent("destroy")
```

What I like about `appendingPathComponent` is not having to specify the `/` separator.